lib/pscom/pscom_priv.h

```diff
@@ -29,7 +29,7 @@
 #include "pscom_gm.h"
 #include "pscom_env.h"
 #include "pscom_precon.h"
-
+#include "../pscom4ivshmem/pscom_ivshmem.h" 
```

Comment:
It is required to include the ivshmem_conn_t type definition. For example, the shm plugin does it the same way. But it could also be restructured according to the  psoib_conn_t.